test/namespace.go

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// namespace.go provides methods to create and delete namespaces.
+
+package test
+
+import (
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// CreateNamespace creates and returns the namespace that was successfully created otherwise error
+func CreateNamespace(clients *Clients, name string) (*v1.Namespace, error) {
```

Comment:
The only client currently ignores the namespace returned, and most use-cases only need the namespace name which is passed. Why return it?

Reply:
Updated to only return error